sections/sections-app/src/webapp/css/sections.css

```diff
@@ -6,7 +6,7 @@
 .selected {width:210px; text-align:right; vertical-align:top;}
 
 .transferButtonTable {text-align:center; font-weight:bold;}
-.studentSectionInfo {font-weight:bold;}
+.studentSectionInfo:last-child {font-weight:bold;}
```

Comment:
This works, but I think ".sectionTable .studentSectionInfo" as the selector makes more sense semantically.